TestCases/rans/naca0012/turb_NACA0012_sst_expliciteuler.cfg

```
@@ -0,0 +1,77 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Testing explicit solution of flow and turbulence equations
+% Author: Max Aehle
+% Institution: TU Kaiserslautern
+% Date: Nov 17th, 2021                                                         %
+% File Version 7.2.1 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+SOLVER= RANS
+KIND_TURB_MODEL= SST
+MATH_PROBLEM= DIRECT
+RESTART_SOL= NO
+READ_BINARY_RESTART= NO
+
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+MACH_NUMBER= 0.15
+AOA= 10.0
+FREESTREAM_TEMPERATURE= 300.0
+REYNOLDS_NUMBER= 6.0E6
+REYNOLDS_LENGTH= 1.0
+ 
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+REF_LENGTH= 1.0
+REF_AREA= 1.0
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+MARKER_HEATFLUX= ( airfoil, 0.0 )
+MARKER_FAR= ( farfield )
+MARKER_PLOTTING= ( airfoil )
+MARKER_MONITORING= ( airfoil )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+NUM_METHOD_GRAD_RECON= LEAST_SQUARES
+CFL_NUMBER= 0.1
+MAX_DELTA_TIME= 1E10
+ITER= 99999
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+CONV_NUM_METHOD_FLOW= ROE
+USE_VECTORIZATION= YES
+MUSCL_FLOW= YES
+SLOPE_LIMITER_FLOW= NONE
+TIME_DISCRE_FLOW= EULER_EXPLICIT
+
+% -------------------- TURBULENT NUMERICAL METHOD DEFINITION ------------------%
+CONV_NUM_METHOD_TURB= SCALAR_UPWIND
+MUSCL_TURB= NO
+SLOPE_LIMITER_TURB= NONE
+TIME_DISCRE_TURB= EULER_EXPLICIT
+CFL_REDUCTION_TURB= 1.0
+
+% --------------------------- CONVERGENCE PARAMETERS --------------------------%
+CONV_FIELD= RMS_DENSITY
+CONV_RESIDUAL_MINVAL= -12
+CONV_STARTITER= 10
+CONV_CAUCHY_ELEMS= 100
+CONV_CAUCHY_EPS= 1E-6
+
+% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
+MESH_FILENAME= n0012_113-33.su2
+MESH_FORMAT= SU2
+MESH_OUT_FILENAME= mesh_out.su2
+SOLUTION_FILENAME= solution_flow_sst_expliciteuler.dat
+TABULAR_FORMAT= CSV
+CONV_FILENAME= history
+RESTART_FILENAME= restart_flow.dat
+VOLUME_FILENAME= flow
+SURFACE_FILENAME= surface_flow
+OUTPUT_WRT_FREQ= 10000
+SCREEN_OUTPUT= (INNER_ITER, RMS_DENSITY, RMS_TKE, RMS_DISSIPATION, LIFT, DRAG, LINSOL_RESIDUAL)
+OUTPUT_FILES= (RESTART_ASCII, PARAVIEW, SURFACE_PARAVIEW)
```

```suggestion
% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
MESH_FILENAME= n0012_113-33.su2
MESH_FORMAT= SU2
%
SCREEN_OUTPUT= (INNER_ITER, RMS_DENSITY, RMS_TKE, RMS_DISSIPATION, LIFT, DRAG, LINSOL_RESIDUAL)
%
TABULAR_FORMAT= CSV
CONV_FILENAME= history
%
OUTPUT_FILES= (RESTART_ASCII, PARAVIEW, SURFACE_PARAVIEW)
OUTPUT_WRT_FREQ= 10000
RESTART_FILENAME= restart_flow.dat
SOLUTION_FILENAME= solution_flow_sst_expliciteuler.dat
VOLUME_FILENAME= flow
SURFACE_FILENAME= surface_flow
```
Just a little reshuffling (and mesh_out removed), I prefer to have the output options in a somewhat seperated screen-history-volume order